packages/renderer/src/lib/ui/ConnectionStatus.svelte

```diff
@@ -11,35 +11,35 @@ const roundIconStyle = 'my-auto w-3 h-3 rounded-full';
 const labelStyle = 'my-auto ml-1 font-bold text-[9px]';
 const statusesStyle = new Map<string, connectionStatusStyle>([
   [
-    'started',
```

Comment:
These styles are also used by some providers. Maybe this could break something.
E.g i nthe PreferencesResourcesRendering.svelte there is `<ConnectionStatus status="{container.status}" />` and status is of type `export type ProviderConnectionStatus = 'started' | 'stopped' | 'starting' | 'stopping' | 'unknown';`

Reply:
Argh, you are right.

You know what we could do instead is just simply edit the svelte file for appearances, but keep the underlying "started" "stopped" etc.

I'll update this PR.